src/main/groovy/com/palantir/gradle/docker/DockerRunExtension.groovy

```diff
@@ -93,14 +94,22 @@ class DockerRunExtension {
     private void setEnvSingle(String key, String value) {
         this.env.put(checkNotNull(key, "key"), checkNotNull(value, "value"))
     }
-
-    public void env(Map<String,String> env) {
-        this.env = ImmutableMap.copyOf(env)
-    }
+	
+	public void env(Map<String,String> env) {
```

Comment:
Please fix the indentation in this file

Reply:
Fixed, sorry it was a spaces - tabs issue.